air/src/trace/mod.rs

```diff
@@ -44,6 +44,9 @@ pub const STACK_TRACE_OFFSET: usize = DECODER_TRACE_RANGE.end;
 pub const STACK_TRACE_WIDTH: usize = 19;
 pub const STACK_TRACE_RANGE: Range<usize> = range(STACK_TRACE_OFFSET, STACK_TRACE_WIDTH);
 
+/// Label for log_precompile transcript state messages on the virtual table bus.
+pub const LOG_PRECOMPILE_LABEL: u8 = 200;
```

Comment:
I need to double check, but we should be able to use the `LOG_PRECOMPILE` opcode here? The issue is that the messages are shared with all chiplet messages which use the chiplet selector labels.